src/style.css

```diff
@@ -0,0 +1,97 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
```

Comment:
You need to add font-family to your selectors to avoid failed tests

Hint: add `font-family` to `html` tag. For `<inputs>` you can `inherit` your fonts